### ISSUE TITLE
Hammer 3x3 not mining harder blocks around the starter block when it has the ability to do so.

### ISSUE THREAD

**Issue by HeavyDeception** [SilentChaos512/Silent-Gear#646]
## What Should Be Added or Changed?

The change I would like to ask about is how hammers will not mine harder blocks around the block you first mine.  This kind of makes it annoying when mods and even vanilla add different blocks at lower levels.  Even when your hammer is a tire 4 or more it still mines this way. Is there no way to change this?

## Why Should This Be Done?

QOL and for the sake of improvement to the mod as other mods seem to work around this. AKA Thermal, Just Hammers, Industrial Foregoing and, Immersive Engineering. To name a few.

**Comment by JonKosh** [comment]
I agree this needs to be added , Either as just base or as a trait. I have been playing ATM9 and its very annoying. Hopefully this will be added soon.
